src/pages/ReviewPage.js

```diff
@@ -33,25 +33,17 @@ class ReviewPage extends React.Component {
   }
 
   translateReason(reason) {
-    if (reason) {
-      switch (reason[0]) {
-      case 'yes':
-        return <Trans>Yes</Trans>
-      default:
-        return <Trans>No</Trans>
-    }} else {
+    if (reason || reason === 'yes') {
+      return <Trans>Yes</Trans>
+    } else {
       return <Trans>No</Trans>
     }
   }
 
   translate(reason) {
```

Comment:
translate is not used, we should just delete the code

Reply:
Removed!